corporate/tests/test_stripe.py

```diff
@@ -6350,6 +6350,68 @@ def test_upgrade_user_to_fixed_price_monthly_basic_plan(self, *mocks: Mock) -> N
         ]:
             self.assert_in_response(substring, response)
 
+    @responses.activate
+    @mock_stripe()
+    def test_delete_configured_fixed_price_plan_offer(self, *mocks: Mock) -> None:
+        self.login("iago")
+
+        self.add_mock_response()
+        with time_machine.travel(self.now, tick=False):
+            send_server_data_to_push_bouncer(consider_usage_statistics=False)
+
+        self.assertFalse(CustomerPlanOffer.objects.exists())
+
+        # Set fixed_price without configuring required_plan_tier.
+        annual_fixed_price = 1200
+        result = self.client_post(
+            "/activity/remote/support",
+            {"remote_realm_id": f"{self.remote_realm.id}", "fixed_price": annual_fixed_price},
+        )
+        self.assert_in_success_response(["Required plan tier should not be set to None"], result)
```

Comment:
>   Set fixed_price without configuring required_plan_tier.

We can skip this code block. This was included in one of the test for coverage. So, I think we should avoid repetition.

Reply:
Removed, thanks!